privaterelay/plans.py

```diff
@@ -543,17 +545,24 @@ def get_bundle_country_language_mapping() -> PlanCountryLangMapping:
     },
     "premium_eu_expansion": {
         "bg": {"bg": "BG"},  # Bulgaria
-        "cy": {"en": "CY"},  # Cyprus
+        "cy": {"el": "CY"},  # Cyprus
         "cz": {"cs": "CZ"},  # Czech Republic / Czechia
         "dk": {"da": "DK"},  # Denmark
         "ee": {"et": "EE"},  # Estonia
         "gr": {"el": "GR"},  # Greece
-        "hr": {"en": "HR"},  # Croatia (language may change from English)
+        "hr": {"hr": "HR"},  # Croatia
         "hu": {"hu": "HU"},  # Hungary
         "lt": {"lt": "LT"},  # Lithuania
-        "lu": {"en": "LU"},  # Luxembourg (language may change from English)
+        "lu": {  # Luxembourg
+            "fr": "LU",
+            "de": "LU",
+            "en": "LU",
+        },
         "lv": {"lv": "LV"},  # Latvia
-        "mt": {"en": "MT"},  # Malta (language may change from English)
+        "mt": {  # Malta
+            "en": "MT",
+            "mt": "MT",
```

Comment:
(comment) Interesting that while these are both official language English is higher up than Maltese. Should we do that for other languages too that have multiple official languages? But defaulting to English to ensure a more consistent experience--we write everything in English first--makes sense to me.

Reply:
Looking at the front-end code, it is only used to distinguish when there are multiple plans for a country, so it doesn't make sense to have several languages map to the same plan, other than expanding the data transfer. I'm going to reduce these to one item.